integrationtests/src/test/java/io/kroxylicious/proxy/KrpcFilterIT.java

```diff
@@ -145,6 +161,39 @@ public void requestFiltersCanRespondWithoutProxying(KafkaCluster cluster, Admin
         }
     }
 
+    @Test
+    public void requestFiltersCanRespondWithoutProxyingDoesntLeakBuffers(KafkaCluster cluster, Admin admin) throws Exception {
+        var config = withDefaultFilters(proxy(cluster))
+                .addToFilters(new FilterDefinitionBuilder("CreateTopicRejectFilter").build());
+
+        try (var tester = kroxyliciousTester(config);
+                var proxyAdmin = tester.admin()) {
+            // loop because System.gc doesn't make any guarantees that the buffer will be collected
+            for (int i = 0; i < 20; i++) {
+                // CreateTopicRejectFilter allocates a buffer and then short-circuit responds
+                assertCreatingTopicThrowsExpectedException(proxyAdmin);
+                // buffers must be garbage collected before it causes leak detection during
+                // a subsequent buffer allocation
+                System.gc();
+            }
+
+            // check no topic created on the cluster
+            Set<String> names = admin.listTopics().names().get(10, TimeUnit.SECONDS);
+            // remove once https://github.com/kroxylicious/kroxylicious-junit5-extension/issues/114 is fixed.
+            names = new HashSet<>(names);
+            names.removeIf(n -> n.startsWith("__org_kroxylicious_testing"));
```

Comment:
[#114](https://github.com/kroxylicious/kroxylicious-junit5-extension/issues/114) is fixed are you still seeing problems related to the quorum test topic?

Reply:
Removed, must have copy pasted a test before the cleanup